core/src/main/java/com/zextras/carbonio/files/dal/dao/User.java

```diff
@@ -19,15 +20,16 @@ public class User {
   private final String fullName;
   private final String email;
   private final String domain;
+  private final UserStatus userStatus;
+  private final UserType userType;
```

Comment:
Maybe here we can use the same convention used in the carbonio-user-management-sdk
```suggestion
  private final UserStatus status;
  private final UserType type;
```